### ISSUE TITLE
Proposal: PostgreSqlConnection.Options - Extend with table details

### ISSUE THREAD

**Issue by snowinmars-debug** [hangfire-postgres/Hangfire.PostgreSql#378]
In my project I applied the hangfire.postgresql nuget package to a custom database. I expect that it will appreciate default psql snake_case style - like efcore - but it does not. So I reverted the database to whatever default hangfire wants.

But it may be possible. It may be possible to run hangfire over a custom database. I can create a pull request, that will allow full schema overriding - table and column names. Because hangfire build string sql queries in runtime, it seems relativy easy to implement. Expect one case: the install scripts. If I want to run it over a custom database, the install scripts should be templated in one way or another.

So, what about this proposal? Am I the only one who wants custom database? I found a [so question](https://stackoverflow.com/questions/27787741/how-to-rename-a-table-inside-a-schema) though.

**Comment by azygis** [comment]
No, sorry, this is not something that's really relevant. I personally don't see the value in having snake case when we use lowercase names either way. Sacrificing performance lost with string formatting on each query, each table is not something that I'd be willing to do if it only makes the schema "pretty". The database should not really be used by anything other than the provider itself.
